modules/local/payload/resources/usr/bin/main.py

```diff
@@ -96,15 +101,48 @@ def rename_file(f, payload, seq_experiment_analysis_dict, date_str):
 
 
 def get_files_info(file_to_upload):
+
+    # Name format for index file: MONSTAR-JP.DO264836.SA626657.targeted-seq.20240926.somatic-germline.cnv.vcf.gz.tbi
+    # Name format for vcf file: MONSTAR-JP.DO264836.SA626657.targeted-seq.20240926.somatic-germline.cnv.vcf.gz
```

Comment:
Please see the name conventions: https://wiki.oicr.on.ca/display/icgcargotech/File+naming+conventions and reference implementation: https://github.com/icgc-argo-workflows/data-processing-utility-tools/blob/main/payload-gen-variant-calling/main.py#L138-L148
Let's be consistent and add the `workflow-short-name` between `date_str` and "somatic-germline".
Btw, should we always hardcode the `somatic_or_germline` to `somatic-germline`?